.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,32 @@
+name: Feature request
+description: Suggest an idea for this project
+labels: [enhancement]
+
+body:
+  - type: textarea
+    id: problem
+    attributes:
+      label: Is your feature request related to a problem? Please describe.
+      description: A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+    validations:
+      required: true
+
+  - type: textarea
+    id: solution
+    attributes:
+      label: Describe the solution you'd like
+      description: A clear and concise description of what you want to happen.
```

Comment:
```suggestion
    id: desired-situation
    attributes:
      label: Desired Situation
      description: |
        What would the user want to happen instead of what is happening?

        Explain it broadly, without going into details of how you would implement the solution.
        
        Don't just think about what would **solve the problem** 💪, but also think about what would **reduce frustration** 🤩.
```